test/testerrorlogger.cpp

```diff
@@ -193,6 +194,34 @@ class TestErrorLogger : public TestFixture {
         }
     }
 
+    void ErrorMessageColorized() const {
+        const bool oDisableColors = gDisableColors;
+        gDisableColors = false;
+        setenv("CLICOLOR_FORCE", "1", 1);
```

Comment:
Thanks. Since it requires you to modify the environment this should be implemented as a Python test instead. This should probably live next to the ones added when `CLICOLOR_FORCE` was introduced.